From 26f36461f66ac62bf6d3f1ef7a1439ab3ed67cf6 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sat, 30 May 2015 14:55:16 -0400
Subject: [PATCH 12/18] nscd: drop selinux/flask.h include

Building nscd w/selinux enabled yields a warning which yields an error:
In file included from selinux.c:32:0:
/usr/include/selinux/flask.h:5:2: error:
 #warning "Please remove any #include's of this header in your source code."

I've done just that and it builds cleanly with libselinux-2.4.

(cherry picked from commit 808696696837b8b8fc858f2e6f8d4e40e26e1308)
---
 ChangeLog      | 4 ++++
 nscd/selinux.c | 1 -
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/ChangeLog b/ChangeLog
index 868d355..079b411 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,7 @@
+2015-07-08  Mike Frysinger  <vapier@gentoo.org>
+
+	* nscd/selinux.c: Delete selinux/flask.h include.
+
 2015-03-02  Andreas Schwab  <schwab@suse.de>
 
 	* elf/Makefile ($(elf-objpfx)runtime-linker.st): Fix typo in
diff --git a/nscd/selinux.c b/nscd/selinux.c
index e433b1b..806763f 100644
--- a/nscd/selinux.c
+++ b/nscd/selinux.c
@@ -29,7 +29,6 @@
 #include <unistd.h>
 #include <sys/prctl.h>
 #include <selinux/avc.h>
-#include <selinux/flask.h>
 #include <selinux/selinux.h>
 #ifdef HAVE_LIBAUDIT
 # include <libaudit.h>
-- 
2.7.1

